src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java

```diff
@@ -28,6 +29,7 @@
   private final CommandLine commandLine;
   private final ADBParser adbParser;
   private Project project;
+  private boolean isTCPEnabled = false;
```

Comment:
You're right, we don't know if the user disabled this connection from command line or not. 
Having just 1 device connected, we can enable TCP every time user connects the new device.
But it's not true for list of devices. Once TCP command called, it refreshes all the connections, and just 1 device at a time will be connected. 
I'll try to figure out how to check if TCP port opened using adb commands.